tools/rosnode/src/rosnode/__init__.py

```diff
@@ -182,7 +182,6 @@ def get_nodes_by_machine(machine):
     @raise ROSNodeException: if machine name cannot be resolved to an address
     @raise ROSNodeIOException: if unable to communicate with master
     """
-    import urlparse
```

Comment:
Already imported at the top of the file so no need to import here. Moreover there is no `urlparse` module in Python 3.